packages/opencensus-propagation-b3/src/b3-format.ts

```diff
@@ -36,21 +32,17 @@ export class B3Format implements Propagation {
    * in the headers, null is returned.
    * @param getter
    */
-  extract(getter: HeaderGetter): SpanContext {
-    if (getter) {
-      let opt = getter.getHeader(X_B3_SAMPLED);
-      if (opt instanceof Array) {
-        opt = opt[0];
-      }
-      const spanContext = {
-        traceId: getter.getHeader(X_B3_TRACE_ID),
-        spanId: getter.getHeader(X_B3_SPAN_ID),
-        options: isNaN(Number(opt)) ? undefined : Number(opt)
-      } as SpanContext;
-
-      return spanContext;
+  extract(getter: HeaderGetter): SpanContext|null {
+    if (!getter) return null;
+    let opt = getter.getHeader(X_B3_SAMPLED);
+    if (opt instanceof Array) {
+      opt = opt[0];
     }
-    return null;
+    return {
+      traceId: getter.getHeader(X_B3_TRACE_ID),
+      spanId: getter.getHeader(X_B3_SPAN_ID),
+      options: isNaN(Number(opt)) ? NOT_SAMPLED_VALUE : Number(opt)
+    } as SpanContext;
```

Comment:
Can the `as SpanContext` be removed since the return type is implied by the function signature?

Reply:
Done.